app/src/main/java/org/wikipedia/bridge/JavaScriptActionHandler.kt

```diff
@@ -28,6 +28,10 @@ object JavaScriptActionHandler {
     }
 
     @JvmStatic
+    fun getTextSelection(): String {
+        return String.format("pagelib.c1.InteractionHandling.getSelectionInfo()")
+    }
+
     fun getOffsets(): String {
```

Comment:
Looks like you removed the `@JvmStatic` by accident.

Reply:
I have updated the PR.

Reply:
Thanks!